assets/dev/js/editor/components/template-library/views/library-layout.js

```diff
@@ -25,13 +25,25 @@ module.exports = elementorModules.common.views.modal.Layout.extend( {
 	},
 
 	getTemplateActionButton: function( templateData ) {
-		var viewId = '#tmpl-elementor-template-library-' + ( templateData.isPro ? 'get-pro-button' : 'insert-button' );
+		var viewId = '#tmpl-elementor-template-library-' + ( 0 !== templateData.accessLevel ? 'get-pro-button' : 'insert-button' );
```

Comment:
Instead of using abstract numbers, it's better to create constants that explain the meaning of each access level